src/WebJobs.Extensions.DurableTask/DurableOrchestrationContext.cs

```diff
@@ -80,7 +80,7 @@ public sealed class DurableOrchestrationContext
         /// <value>
         /// <c>true</c> if the orchestrator function is currently being replayed; otherwise <c>false</c>.
         /// </value>
-        public bool IsReplaying => this.innerContext.IsReplaying;
+        public virtual bool IsReplaying => this.innerContext.IsReplaying;

```

Comment:
Let's move `IsReplaying` into the abstract base class as a virtual property with a default value of `false`.